lib/smart_proxy_acd/acd.rb

```diff
@@ -5,8 +5,18 @@ class NotFound < RuntimeError; end
     # Implement a SmartProxy Plugin
     class Plugin < ::Proxy::Plugin
       plugin 'acd', Proxy::Acd::VERSION
-      http_rackup_path File.expand_path('acd_http_config.ru', File.expand_path('../', __FILE__))
-      https_rackup_path File.expand_path('acd_http_config.ru', File.expand_path('../', __FILE__))
+      http_rackup_path File.expand_path('acd_http_config.ru', __dir__)
+      https_rackup_path File.expand_path('acd_http_config.ru', __dir__)
```

Comment:
It gets unrelated, but with recent Smart Proxy versions (2.3+) you can use
```suggestion
      rackup_path File.expand_path('acd_http_config.ru', __dir__)
```